environments/common/inventory/group_vars/all/nfs.yml

```diff
@@ -15,3 +15,16 @@ nfs_configurations:
     nfs_server: "{{ nfs_server_default }}"
     nfs_export: "/exports/home" # assumes skeleton TF is being used
     nfs_client_mnt_point: "/home"
+
+  - comment: Export /etc/hosts copy from Slurm control node
```

Comment:
Why don't we just have a single directory, /exports/cluster, with the hosts file within that.